bedrock/products/templatetags/misc.py

```diff
@@ -300,33 +302,84 @@ def relay_bundle_countries():
+    price = monthly_when_monthly if plan == "monthly" else float(yearly_when_yearly)
 
-# free is available everywhere
-# email is available according to RELAY_EMAIL_COUNTRY_CODES
-# phone is available according to RELAY_PHONE_COUNTRY_CODES
-# bundle is available according to RELAY_VPN_BUNDLE_COUNTRY_CODES
+    ga_data = (
+        "{"
+        "'id' : '%s',"
+        "'brand' : '%s',"
+        "'plan' : '%s',"
+        "'period' : '%s',"
+        "'price' : '%s',"
+        "'discount' : '%s',"
+        "'currency' : '%s'"
+        "}" % (id, "relay", product, "monthly" if plan == "monthly" else "yearly", str(format(price, ".2f")), str(format(discount, ".2f")), currency)
+    )
+
+    return ga_data
 
 
-def _relay_get_plans(country_code, lang, product):
+def _relay_get_region(product, country_code, lang):
     """
-    Get subscription plan IDs using country_code and page language.
-    Defaults to "US" if no matching country code is found.
-    Each country also has a default language if no match is found.
+    Return the "region" code for this request
     """
 
-    if product == "relay-bundle":
-        country_plans = settings.RELAY_VPN_BUNDLE_COUNTRY_LANG_MAPPING.get(country_code, settings.RELAY_VPN_BUNDLE_COUNTRY_LANG_MAPPING["US"])
-    elif product == "relay-phone":
-        country_plans = settings.RELAY_PHONE_PLAN_COUNTRY_LANG_MAPPING.get(country_code, settings.RELAY_PHONE_PLAN_COUNTRY_LANG_MAPPING["US"])
-    else:  # product == 'relay-email'
-        country_plans = settings.RELAY_EMAIL_PLAN_COUNTRY_LANG_MAPPING.get(country_code, settings.RELAY_EMAIL_PLAN_COUNTRY_LANG_MAPPING["US"])
+    lang = lang.split("-")[0]  # i.e. if it's en-US we only want the en part
 
-    return country_plans.get(lang, country_plans.get("default"))
+    product = product.replace("relay-", "")
+    region_langs = settings.RELAY_PLANS_BY_COUNTRY_AND_LANGUAGE[product].get(country_code)
+
+    # get region that matches requested language, or get first region
+    region = region_langs.get(lang, list(region_langs.values())[0])
+
+    return region
+
+
+def _relay_get_product_region_dict(product, region):
+    """
+    Return a dict for the product's region, containing stripe IDs and currency
+    """
+
+    product = product.replace("relay-", "")
+    product_region_dict = settings.RELAY_STRIPE_PLAN_DATA[product]["countries_and_regions"][region]
+
+    return product_region_dict
+
+
+def _relay_get_plan_id(product, country_code, lang, plan):
+    """
+    Return Stripe Plan ID for matching country, language, plan
+    """
+
+    region = _relay_get_region(product, country_code, lang)
+    dict = _relay_get_product_region_dict(product, region)
+    plan = "monthly_id" if plan == "monthly" else "yearly_id"
+
+    return dict[plan]
+
+
+def _relay_get_currency(product, country_code, lang):
+    region = _relay_get_region(product, country_code, lang)
+    dict = _relay_get_product_region_dict(product, region)
+
+    return dict["currency"]
```

Comment:
Since `dict` is a Python keyword, I'd opt for another variable name if possible.